src/test/java/api/loans/RenewalAPITests.java

```diff
@@ -769,6 +771,24 @@ public void cannotRenewWhenItemIsNotLoanable() {
       hasLoanPolicyNameParameter("Non loanable policy"))));
   }
 
+  @Test
+  public void cannotRenewWhenItemIsDeclaredLost() {
+
+    final IndividualResource smallAngryPlanet = itemsFixture.basedUponSmallAngryPlanet();
+    final IndividualResource jessica = usersFixture.jessica();
+
+    final JsonObject loanJson = loansFixture.checkOutByBarcode(smallAngryPlanet, usersFixture.jessica())
+      .getJson();
+
+    declareItemLost(loanJson);
+
+    final Response response = attemptRenewal(smallAngryPlanet, jessica);
+
+    assertThat(response.getJson(), hasErrorWith(allOf(
+      hasMessage("Item not renewed: item is Declared lost"),
```

Comment:
I may have already asked this before, however I've lost the comment thread.

Has this error message been checked with Emma, as it is structured differently to the others?

Reply:
The message is according to story's requirement. I am waiting the confirmation of product owner.